src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -348,20 +348,20 @@ private static class Logging {
      */
     private boolean streamFilterSet;
 
-    /* Cache LUDCL (Latest User-Defined Class Loader) until completion of read requests.
-     * If true LUDCL/forName results would be cached, true by default starting with Java 8.
+
```

Comment:
Just one blank line at a time, please (line 350 is blank, we don't need line 351).